app/controllers/spectrums_controller.rb

```diff
@@ -349,9 +349,9 @@ def update
     @spectrum = Spectrum.find(params[:id])
     require_ownership(@spectrum)
 
-    @spectrum.title = params[:title] unless params[:title].nil?
-    @spectrum.notes = params[:notes] unless params[:notes].nil?
-    @spectrum.data  = params[:data] unless params[:data].nil?
+    @spectrum.title = params[:spectrum][:title] unless params[:spectrum].nil? || params[:spectrum][:title].nil?
+    @spectrum.notes = params[:spectrum][:notes] unless params[:spectrum].nil? || params[:spectrum][:notes].nil?
+    @spectrum.data  = params[:spectrum][:data] unless  params[:spectrum].nil? || params[:data].nil?
```

Comment:
Wasn't sure where we allow this, actually. not from the HTML UI form... but maybe via the API? Anyways i updated it to match.

Reply:
```
Failure:
SpectrumsControllerTest#test_should_update_spectrum_notes [/home/runner/work/spectral-workbench/spectral-workbench/test/controllers/spectrums_controller_test.rb:264]:
Expected: "Neon spectrum info"
  Actual: "A test spectrum."

bin/rails test test/controllers/spectrums_controller_test.rb:258

.........F

Failure:
SpectrumsControllerTest#test_should_update_spectrum_title [/home/runner/work/spectral-workbench/spectral-workbench/test/controllers/spectrums_controller_test.rb:252]:
Expected: "Really cool spectrum"
  Actual: "One nice spectrum"

bin/rails test test/controllers/spectrums_controller_test.rb:246
```